src/executors/default.yml

```diff
@@ -4,6 +4,6 @@
 description: Preconfigured CircleCI Python image.
 
 docker:
-  - image: noosenergy/circleci:latest
+  - image: noosenergy/circleci:3.11.4
```

Comment:
💡 

Maybe worth to parameterize the `orb` version number to be able to simply switch between docker images?

```
version: 2.1
# config.yml
orbs:
  foo-orb: somenamespace/foo@1
jobs:
  some-job:
    executor: foo-orb/bar  # prefixed executor
      tag: latest
```